games-strategy/dopewars/dopewars-1.5.12.recipe

```diff
@@ -0,0 +1,73 @@
+SUMMARY="Make a fortune dealing drugs on the streets of New York"
+DESCRIPTION="dopewars is a free Unix/Win32 rewrite of a game originally based \
+on \"Drug Wars\" by John E. Dell. The idea of dopewars is to deal in drugs on \
+the streets of New York, amassing a huge fortune and paying off the loan \
+shark, while avoiding the ever-annoying police. The Unix/Win32 rewrite, as \
+featuring a so-called \"antique\" mode which closely follows the original, \
+introduces new features such as the ability to take part in multi-player \
+games. dopewars aims to be highly configurable, and what you can't change in \
+the configuration files you can change by poking around in the source, which \
+is freely available under the terms of the GNU General Public License.
+
+ADVISORY: dopewars is not recommended for children below 16 without parental \
+supervision. We also do not condone real-life drug dealing."
+HOMEPAGE="https://dopewars.sourceforge.io/"
+COPYRIGHT="1999-2005 dopewars developers"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/dopewars/dopewars-$portVersion.tar.gz"
+CHECKSUM_SHA256="23059dcdea96c6072b148ee21d76237ef3535e5be90b3b2d8239d150feee0c19"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+GLOBAL_WRITABLE_FILES="
+	var/dopewars.sco keep-old
+	"
+
+PROVIDES="
+	dopewars = $portVersion
+	cmd:dopewars
+	"
+REQUIRES="
+	haiku
+	lib:libglib
+	lib:libiconv
+	lib:libintl
+	lib:libncurses
+	lib:libSDL_1.2
+	lib:libSDL_mixer_1.2
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libglib
+	devel:libiconv
+	devel:libintl
+	devel:libncurses
+	devel:libSDL
+	devel:libSDL_mixer
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc
+	cmd:make
+	"
+
+PATCH()
+{
+	sed -i '/ac_lib/ s/socket/network/g' configure
+}
+
+BUILD()
+{
+	runConfigure --omit-dirs "dataRootDir docDir" ./configure --build=x86
+	sed -i '/dopewars_DEPENDENCIES/ s/-lintl -liconv//' src/Makefile
```

Comment:
The sed command doesn't really add anything here (no changes to the Makefile that shows when linking dopewars)
using export * pulls in -lintl and -liconv at linking time
```diff
+	export LIBS="-lintl -liconv"
	runConfigure --omit-dirs "dataRootDir docDir" ./configure --build=x86
-	sed -i '/dopewars_DEPENDENCIES/ s/-lintl -liconv//' src/Makefile
```

Reply:
I mean, it's to prevent it from failing because it lists `-lintl` and `-liconv` as file dependencies that need to be made. But for some reason it doesn't happen when they're exported so I guess that's cleaner. Or maybe I misunderstood something?

Reply:
running configure without the options also fill in the needed dependencies, but I've seen it before that INTLLIBS=-lintl doesn't get pulled in at linking time, hence using LIBS=* works

Reply:
To clarify, the `sed` command was because `-lintl` and `-liconv` were listed in `dopewars_DEPENDENCIES`, which is for files that need to be made. So obviously it fails, saying "No rule to make target '-lintl', needed by 'dopewars'". But it doesn't get listed there when they're in `LIBS`, so it's not needed with that, but it wasn't useless earlier! :)